Automattic/simplenote-macos
# Shortcuts: Add open new note intent

### Fix
So the very first shortcut to add is one that simply opens Simplenote and creates a new note.  This is a basic proof of concept that the intents and shortcut handling works.

So in this PR you can create a shortcut "Open New Note" that will launch Simplenote and then create a new note.

Relies on #1159 
### Test
1. Build the app.  
2. Go to the shortcuts app, click on the plus button to add a new shortcut, search for Simplenote, select it, choose Open New Note
3. Run the shortcut.

- [ ] confirm that Simplenote opens and that a new note is created

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
***(Required)*** Add a concise statement to `RELEASE-NOTES.txt` if the changes should be included in release notes. Include details about updating the notes in this section. For example:
> These changes do not require release notes.
